tests/integration/ceph_upgrade_test.go

```diff
@@ -380,7 +380,7 @@ func (s *UpgradeSuite) verifyOperatorImage(expectedImage string) {
 	// verify that the operator spec is updated
 	version, err := k8sutil.GetDeploymentImage(context.TODO(), s.k8sh.Clientset, systemNamespace, operatorContainer, operatorContainer)
 	assert.NoError(s.T(), err)
-	assert.Contains(s.T(), version, "rook/ceph:"+expectedImage)
+	assert.Contains(s.T(), "docker.io/rook/ceph:"+expectedImage, version)
```

Comment:
Maybe we could restore this to `Equal`? I changed to contain because that would fix the ci, but turns out I did miss some stuff.
```suggestion
	assert.Equal(s.T(), "docker.io/rook/ceph:"+expectedImage, version)
```